cypress/pages/workflow.ts

```diff
@@ -126,12 +128,16 @@ export class WorkflowPage extends BasePage {
 				win.preventNodeViewBeforeUnload = preventNodeViewUnload;
 			});
 		},
-		addInitialNodeToCanvas: (nodeDisplayName: string, { keepNdvOpen } = { keepNdvOpen: false }) => {
+		addInitialNodeToCanvas: (nodeDisplayName: string, opts?: { keepNdvOpen?: boolean, action?: string }) => {
 			this.getters.canvasPlusButton().click();
 			this.getters.nodeCreatorSearchBar().type(nodeDisplayName);
 			this.getters.nodeCreatorSearchBar().type('{enter}');
-			if (keepNdvOpen) return;
-			cy.get('body').type('{esc}');
+			if (opts?.action) {
+				nodeCreator.getters.getCreatorItem(opts.action).click();
+			}
+			else if (!opts?.keepNdvOpen) {
+				cy.get('body').type('{esc}');
+			}
```

Comment:
We could simplify this by calling the method below which already supports this behaviour:
```ts
addInitialNodeToCanvas: (nodeDisplayName: string, opts?: { keepNdvOpen?: boolean, action?: string }) =>
			this.getters.canvasPlusButton().click();

			this.actions.addNodeToCanvas(nodeDisplayName, false, opts?.keepNdvOpen, opts?.action);
		}
```

Reply:
tried this but some tests started failing.. maybe we can keep this as for now